### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by Basia2B**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Basia2B/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (Basia2B) [inline ReadMe.md:1]: Making a line note.

Comment (Basia2B) [inline ReadMe.md:1]: comment with fix status !fixed

Comment (Basia2B) [inline ReadMe.md:1]: comment !fix

Comment (Basia2B) [inline ReadMe.md:1]: !fixed

**Pull request merged**